app/admin.go

```diff
@@ -24,7 +24,7 @@ var latestVersionCache = cache.NewLRU(cache.LRUOptions{
 	Size: 1,
 })
 
-func (s *Server) GetLogs(page, perPage int) ([]string, *model.AppError) {
+func (s *Server) GetLogsOld(page, perPage int) ([]string, *model.AppError) {
```

Comment:
Rather than renaming this, we can actually do something like:
```suggestion
func (s *Server) GetLogs(page, perPage int, options ...model.GetLogOption) ([]string, *model.AppError) {
```
This way we wouldn't need to update every reference of this method.

Reply:
I think keeping them as separate methods makes more sense than adding an arbitrary parameter that decides which code to run. I'm going to be changing the name of the new function and leaving the old one intact so that it's less confusing.